tests/RandomHelpersT.cpp

```diff
@@ -41,10 +47,98 @@ TEST_CASE("[Random] Fast random generation")
     unsigned numGenerations = 1000;
     unsigned numDivisions = 128;
 
-    auto& test = randomTest<fast_rand, fast_real_distribution<float>>;
+    auto& test = uniformRandomTest<fast_rand, fast_real_distribution<float>>;
 
     REQUIRE(test(0.0f, 1.0f, numGenerations, numDivisions));
     REQUIRE(test(-1.0f, 1.0f, numGenerations, numDivisions));
     REQUIRE(test(0.0f, 123.0f, numGenerations, numDivisions));
     REQUIRE(test(-123.0f, 0.0f, numGenerations, numDivisions));
 }
+
+/**
+ * @brief Check the behavior of a gaussian real random generator
+ *
+ * - compute a histogram of random generations
+ * - compare it against the standard library normal distribution
+ */
+template <unsigned Quality>
+static bool gaussianRandomTest(double mean, float variance, size_t numGen, size_t histSize, double maxAbsErr)
+{
+    fast_gaussian_generator<float, Quality> gen(mean, variance);
+
+    std::mt19937_64 prng;
+    std::normal_distribution<float> dist(mean, variance);
+
+    // the tested bounds
+    const double min = -1.0 + mean;
+    const double max = +1.0 + mean;
+
+    // generate, quantify, count occurrences
+    std::vector<size_t> counts(histSize);
+    for (size_t i = 0; i < numGen; ++i) {
+        double value = gen();
+        double normalized = (value - min) / (max - min);
```

Comment:
Since the normal distribution has unbounded support I am not sure that this normalization makes alot of sense. This value could get negative (with low probability depending on the variance and mean, but still).